### PR TITLE
Improve

### PR DESCRIPTION
**Pull request by MayankLakhotia**
code work faster

### CODE REVIEW

**Comment by MayankLakhotia** [comment]
code work faster and take less space

**Comment by mfrank37** [comment]
Hi @MayankLakhotia ,
Thanks for your will to contribute, welcome.

I agree the `endl` isn't necessary. Your code is spacious and harder to read. 
Also you shouldn't remove the `!` exclamation from the message.

```code work faster and take less space``` by you. I don't see how your changes do this.

File: hello.cpp [hunk 1]
```diff
@@ -3,7 +3,11 @@
 using namespace std;
 
 int main()
+
```
Comment (mfrank37) [inline hello.cpp:6]: extra space

File: hello.cpp [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
 {
-    cout << "Hello World!" << endl;
+
```
Comment (mfrank37) [inline hello.cpp:8]: extra space

File: hello.cpp [hunk 3]
```diff
@@ -3,7 +3,11 @@
...
+    cout << "Hello World";
+
```
Comment (mfrank37) [inline hello.cpp:10]: extra space unnecessary.

File: hello.cpp [hunk 4]
```diff
@@ -3,7 +3,11 @@
...
     return 0;
+
```
Comment (mfrank37) [inline hello.cpp:12]: extra space unnecessary.

File: hello.cpp [hunk 5]
```diff
@@ -3,7 +3,11 @@
 using namespace std;
 
 int main()
+
 {
-    cout << "Hello World!" << endl;
+
+    cout << "Hello World";
```
Comment (mfrank37) [inline hello.cpp:9]: missing `!` from the message.

**Review by mfrank37 — CHANGES_REQUESTED** [review]
Needs more readable code, and `!`

**Comment by MayankLakhotia** [comment]
Bro I am just trying pull request

**Pull request closed without merging**